examples/with-vitest/vite.config.ts

```diff
@@ -23,11 +22,8 @@ export default defineConfig({
     // isolate: false,
   },
   plugins: [solid()],
-  build: {
-    target: "esnext",
-    polyfillDynamicImport: false
-  },
+  
   resolve: {
-    conditions: ["development", "browser"]
-  }
+		conditions: process.env["VITEST"] ? ["browser"] : [],
```

Comment:
why not use `process.env.VITEST` like [official docs](https://vitest.dev/guide/#configuring-vitest)?

Reply:
Basically: `process.env` is typed via an index-signature in `@types/node`. I'm using very strict ts-settings. TS forces me to :)

References: 
- https://www.typescriptlang.org/docs/handbook/2/indexed-access-types.html
- https://www.typescriptlang.org/tsconfig#noPropertyAccessFromIndexSignature